src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -540,8 +541,31 @@ public uint GetPaginationLimit(int? first)
     /// <summary>
     /// Retrieve the value of LogLevel from runtime
     /// </summary>
-    public Level? LogLevelValue()
+    public ExtendedLogLevel? IsLogLevel()
```

Comment:
Is log level what?
we typically use "Is" "Has" etc for booleans. e.g. "IsDeveloperMode() -> true/false". You're just returning the value of `Runtime?.LogLevel?.Value;` here.

Is this function really necessary? could this be baked into your function `GetConfiguredLogLevel()`? This was brought up in this [discussion ](https://github.com/Azure/data-api-builder/pull/2359/files/c15667ba34010df8932732584512de4d8ffead13..a8212a6bccd278d07985f0ddf70b944369763209#r1747557812) but I didn't see any further responses there.

Reply:
Changing IsLogLevel to IsLogLevelNull, so it will only return true/false depending on whether `Runtime?.LogLevel?.Value` is null. So that it can be used in the `RuntimeConfigLoaderJsonDeserializerTests`